examplesv1/src/createAndUpdate.ts

```diff
@@ -0,0 +1,220 @@
+import * as Kilt from '@kiltprotocol/sdk-js'
+import { Crypto } from '@kiltprotocol/utils'
+import { mnemonicGenerate, randomAsU8a } from '@polkadot/util-crypto'
+
+async function createDid() {
+
+  // Create a connection to a KILT blockchain.
+  // let api = await Kilt.connect('wss://peregrine.kilt.io/')
+  let api = await Kilt.connect('ws://localhost:9944/')
```

Comment:
maybe just use a descriptive const?
```suggestion
  let api = await Kilt.connect(FULL_NODE_WEB_SOCKET_URL)
```